.travis.yml

```diff
@@ -47,6 +47,7 @@ env:
     - TRAVIS_FLAVOR=cassandra_nodetool FLAVOR_VERSION=2.1.14
     - TRAVIS_FLAVOR=cassandra_nodetool FLAVOR_VERSION=2.2.10
     - TRAVIS_FLAVOR=couch FLAVOR_VERSION=1.6.1
+    - TRAVIS_FLAVOR=couch FLAVOR_VERSION=2.0-dev
```

Comment:
I don't have any issue with this, just curious why you picked the `2.0-dev` over `2.0` or `2.0.0`

Reply:
`2.0-dev` is a special developers version that build a 3 nodes cluster. Otherwise I guess we'd have to do it ourselves as part of the tests setup.

Reply:
works for me!